examples/null_example/src/null_example.ts

```diff
@@ -0,0 +1,63 @@
+import { int, Query, Update, Variant } from 'azle';
+
+type partially_null_object = {
```

Comment:
The convention for type names is to have them by uppercase camelcase, so this would be `PartiallyNullObject`